dirty_cat/minhash_encoder.py

```diff
@@ -127,27 +135,30 @@ def get_hash(self, string):
         array, shape (n_components, )
             The encoded string, using specified encoding scheme.
         """
-        if self.hashing == 'fast':
-            if self.minmax_hash:
-                assert self.n_components % 2 == 0,\
-                       "n_components should be even when minmax_hash=True"
-                return np.concatenate([ngram_min_hash(string, self.ngram_range,
-                                                      seed, return_minmax=True)
-                                      for seed in range(self.n_components//2)])
-            else:
-                return np.array([ngram_min_hash(string, self.ngram_range, seed)
-                                for seed in range(self.n_components)])
+        if self.minmax_hash:
+            return np.concatenate([ngram_min_hash(string, self.ngram_range,
+                                   seed, return_minmax=True)
+                                   for seed in range(self.n_components//2)])
+        else:
+            return np.array([ngram_min_hash(string, self.ngram_range, seed)
+                            for seed in range(self.n_components)])
 
-        elif self.hashing == 'murmur':
-            assert not(self.minmax_hash),\
-                   "minmax_hash not implemented with murmur"
-            return self.minhash(
+    def get_murmur_hash(self, string):
+        """
+        Encode a string with murmur hashing function.
+        murmur only supports min_hash encoding.
+        Parameters
+        ----------
+        string : str
+            The string to encode.
+        Returns
+        -------
+        array, shape (n_components, )
+            The encoded string, using specified encoding scheme.
+        """
+        return self.minhash(
```

Comment:
When I see a function that small, I find that we have one level of indirection too much. Can we just inline the "minhash" method in here and remove it?

Reply:
fixed